_rules/aria-state-or-property-permitted-5c01ea.md

```diff
@@ -49,81 +50,65 @@ _There are no major accessibility support issues known for this rule._
 
 #### Passed Example 1
 
-`aria-pressed` state is supported for role `button`.
+The `aria-pressed` [state][] is [supported][] for the [semantic role][] `button` which is the [explicit role][] of this `div` element.
 
 ```html
 <div role="button" aria-pressed="false">My button</div>
 ```
 
 #### Passed Example 2
 
-`aria-pressed` state is supported for `role=button` that is the [implicit role](#implicit-role) for `button` element.
+The `aria-pressed` [state][] is [supported][] for the [semantic role][] `button` which is the [implicit role][] for `button` elements.
 
 ```html
 <button aria-pressed="false">My button</button>
 ```
 
 #### Passed Example 3
 
-Global state that is supported by all base markup elements.
+The `aria-busy` [state][] is a [global][] [state][] that is [supported][] by all elements, even without any [semantic role][].
 
 ```html
 <div aria-busy="true">My busy button</div>
 ```
 
 #### Passed Example 4
 
-`aria-label` state is inherited for role `button`
+The `aria-label` [state][] is [inherited][] for the [semantic role][] `button`.
 
 ```html
 <div role="button" aria-label="OK"></div>
 ```
 
 #### Passed Example 5
 
-`aria-checked` state is required for role `aria-checkbox`
+The `aria-checked` [state][] is [required][] for the [semantic role][] `aria-checkbox`.
 
 ```html
 <div role="checkbox" aria-checked="false">My checkbox</div>
 ```
 
 #### Passed Example 6
 
-`aria-controls` property is supported for role `combobox`
+The `aria-controls` [property][] is [supported][] for the [semantic role][] `combobox`.
 
 ```html
 <div role="combobox" aria-controls="id1">My combobox</div>
 ```
 
-#### Passed Example 7
-
-WAI-ARIA states and properties with empty value are also applicable to this rule
-
-```html
-<div role="combobox" aria-controls>My combobox</div>
-```
-
-#### Passed Example 8
-
-WAI-ARIA states and properties with empty value, specified as an empty string, are also applicable to this rule
-
-```html
-<div role="combobox" aria-controls="">My combobox</div>
-```
```

Comment:
I removed these two examples because they are essentially failing the "SC" (except it's not a WCAG SC, but that's the same idea) and we normally avoid Passed Examples that fail their Accessibility Mapping.

Reply:
Perhaps there are better examples for the situation, perhaps for something like a slider, where the required states and properties have defined default values. 

Though my minimal understanding is that such states and properties don't need to be added in the code at all if the default values are being used. Is that why you are removing these examples?

Reply:
Can you do this in a different PR?

Reply:
#1183